clients/drcachesim/tracer/tracer.cpp

```diff
@@ -289,7 +296,7 @@ event_bb_analyze_case(void *drcontext, void *tag, instrlist_t *bb, bool for_trac
                       bool translating, uintptr_t mode, void *user_data,
                       void *orig_analysis_data, void **analysis_data)
 {
-    if (mode == BBDUP_MODE_TRACE) {
+    if (mode == BBDUP_MODE_TRACE || mode == BBDUP_MODE_L0_FILTER) {
```

Comment:
If the helper takes in the already-load-acquired mode it can be used here too and below

Reply:
Done.